### ISSUE TITLE
camera.snapshot doesn't work anymore for Tapo C200

### ISSUE THREAD

**Issue by mitzu2250** [JurajNyiri/HomeAssistant-Tapo-Control#180]
### Description

I noticed that lately, the camera.snapshot is not generating a valid photo for Tapo C200 camera. A file is being saved but it has 0 kb. camera.record still works good and also camera.snapshot works for Tapo C210.

### Reproduction Steps

1. Go to SERVICES
2. Select Camera: Take Snapshot
3. Select Tapo C200 camera
4. Input desired file location (Filename)
5. Press CALL SERVICE

### Expected behavior

File is being saved properly, it can be opened and it contains the image captured by the camera.

### If applicable, add error logs.

No error logs are being generated.

### Device Firmware

1.1.16 Build 211209 Rel.37726n(4555)

### Using stream component

Yes

### Does camera work via official integrations?

N/A

### Camera has all attributes filled out in developer tools

Yes

### HASS Environment

VMware Workstation

### Search for similar issues

Yes

### Additional information

About two weeks ago it still worked. Something happened in the meantime. There are some updated that Home Assistant had but I imagine that it should have also broke the other camera, not just this one.

**Comment by JurajNyiri** [comment]
Please try following:

- Restart camera and try
- Restart Home Assistant and try
- Restart Home Assistant and try
- Restart camera and try
- Restart Home Assistant and try
- Try with stream component turned off
- Try with https://www.home-assistant.io/integrations/onvif/ instead

**Comment by mitzu2250** [comment]
Hello,

I have tried doing a reboot from Home Assistant, this didn't worked.
Tried restarting Home Assistant and this also didn't worked.

But then I tried restarting the camera from Tapo mobile App, and this worked.

For some reason, even though my camera restarted several times due to power outage in the last days, it didn't really 'restarted' until I used the RESTART CAMERA button from the app.

Thank you for your support. 
This can be closed, hopefully will help others with this issue in the future.
